### PR TITLE
Implemented static UI of order sync (#22j85u2)

### PR DESCRIPTION
**Pull request by azkyakhan**

### CODE REVIEW

**Comment by azkyakhan** [comment]
orders

File: src/components/BatchModal.vue [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  </ion-header>
+
+  <ion-content>
+    <ion-item>
+      <ion-label position="fixed">Batch name</ion-label>
+      <ion-input placeholder="Batch 2" />
+    </ion-item>
+    <ion-item>
+      <ion-label position="fixed">{{ $t("Schedule") }}</ion-label>
+      <ion-input type="date" />
```
Comment (dt2patel) [inline src/components/BatchModal.vue:20]: I think we can use Ionic's new date picker component here.

File: src/components/DurationPopover.vue [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+export default defineComponent({
+  name: 'DurationPopover',
+  components: {
+    IonSelect,
+    IonSelectOption
+  },
+ setup(){
+    const customPopoverOptions: any = {
+      header: 'Title',
```
Comment (dt2patel) [inline src/components/DurationPopover.vue:23]: Use title from Figma files

**Review by dt2patel — APPROVED** [review]

**Review by dt2patel — CHANGES_REQUESTED** [review]
Few changes

File: src/components/BatchModal.vue (hunk 1 above)
Comment (azkyakhan) [reply src/components/BatchModal.vue]: Ok sir

File: src/components/DurationPopover.vue (hunk 2 above)
Comment (azkyakhan) [reply src/components/DurationPopover.vue]: Ok

**Review by dt2patel — APPROVED** [review]
looks good

**Pull request merged**